### ISSUE TITLE
Play sound feedback when listening the events

### ISSUE THREAD

**Issue by niw** [niw/HapticKey#7]
In the comment of #4, @chrisballinger brings up that it would be better to have a sound feedback instead of haptic feedback. This is an issue to memo this request.

**Comment by munsuri** [comment]
I'm down for that one too. I was planning to add it myself after the one key event.

**Comment by munsuri** [comment]
Actually, it was quicker than expected to implement audio feedback with [AVAudioPlayer](https://developer.apple.com/documentation/avfoundation/avaudioplayer). I've used a shorter version (quiet start and end removed) of [this sound](https://freesound.org/people/bigmonmulgrew/sounds/378085/). Now, the sound intensity can be controlled (or not) as well through that API.

**Comment by muescha** [comment]
@munsuri you should create a pull request with your modification

**Comment by niw** [comment]
Sorry, I accidentally hit “Close issue” button, please ignore the last activities.

**Comment by chrisballinger** [comment]
If anyone wants to play around with the old Mac OS 9 Platinum sounds for nostalgia purposes, I extracted them from the resource fork of the original sound set I found somewhere online. I'm a fan of `00548 "wopn".aiff`, but there are a number of other good ones.

[Mac OS 9 Platinum Sounds.zip](https://github.com/niw/HapticKey/files/1675884/Mac.OS.9.Platinum.Sounds.zip)

**Comment by niw** [comment]
Okay, so I made a change to support sound effect 4cbef9faa7b6d1667911ac743a8d947538469cec and some other features on `wip/add_sound_effect_and_screen_flash` branch.
Since there are some concerns so didn’t merge this into `master` yet, once it’s solved I’ll post a new release.

@chrisballinger if you want to use that sound, you can add that sound as a resource to the project (Simply drag and drop to Resources group,) and modify line 45 of 4cbef9faa7b6d1667911ac743a8d947538469cec to `[[HTKSystemSound alloc] initWithPath:[[NSBundle mainBundle] pathForResource:@"wopn" ofType:@"aiff"]`.
Due to license, I think it’s not good idea to have that sound file in the repository, but, you can at least try it now. 😉

**Comment by chrisballinger** [comment]
Yeah I agree about the licensing, probably not a good idea to ship those by default. Perhaps a few royalty free default sounds could be included, selectable via Sounds sub-menu. Then you could drop external sound files into `~/Library/Application Support/HapticKey/Sounds/` and enumerate them within the sub-menu along with the bundled sounds. I'd submit a PR for that once things settle down a bit.

**Comment by niw** [comment]
@chrisballinger That approach sounds good!
